src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -253,6 +254,22 @@ TEST_F(PolicyManagerImplTest, LoadPT_SetPT_PTIsLoaded) {
   app_policies["1234"]["steal_focus"] = Json::Value(true);
   app_policies["1234"]["certificate"] = Json::Value("sign");
 
+  app_policies["device"] = Json::Value(Json::objectValue);
+  app_policies["device"]["keep_context"] = Json::Value(false);
```

Comment:
@AGaliuzov Why should these values be omitted? Removing "device" from app policies or its keep_context, default_hmi, or steal_focus values causes the policy table to be considered invalid.


Results if values are removed:
```
policy_manager_impl_test.cc:277: Failure
Value of: IsValid(update)
  Actual: false ( - table.policy_table.app_policies.device: object is not initialized
 - table.policy_table.app_policies.device.groups: object is not initialized
 - table.policy_table.app_policies.device.priority: value is not initialized
 - table.policy_table.app_policies.device.default_hmi: value is not initialized
 - table.policy_table.app_policies.device.keep_context: value is not initialized
 - table.policy_table.app_policies.device.steal_focus: value is not initialized
)
```